ephemeralkey.go

```diff
@@ -0,0 +1,42 @@
+package stripe
+
+import "encoding/json"
+
+// EphemeralKeyParams is the set of parameters that can be used when creating
+// an ephemeral key.
+// For more details see https://stripe.com/docs/api#ephemeral_keys.
+type EphemeralKeyParams struct {
+	Params
+	Customer      string
+	StripeVersion string
+}
+
+// EphemeralKey is the resource representing a Stripe ephemeral key.
+// For more details see https://stripe.com/docs/api#ephemeral_keys.
+type EphemeralKey struct {
+	ID                string `json:"id"`
+	Created           int64  `json:"created"`
+	Expires           int64  `json:"expires"`
+	Live              bool   `json:"livemode"`
+	AssociatedObjects []struct {
+		ID   string `json:"id"`
+		Type string `json:"type"`
+	} `json:"associated_objects"`
+	RawJSON []byte
```

Comment:
Actually, one other ask here: would you mind documenting the `RawJSON` type field here? I don't think it's intuitive as to why it's needed, and I'm a little afraid that it might confuse people as they're reading the code (like as to why this model has it, but no others do).